packages/components/datetime/DateTime.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: 'DateTime'
+type: 'component'
+status: 'stable'
+slug: /components/datetime/
```

Comment:
Needs to be the same slug as set on the gatsby config file, otherwise the link on the menu will point to the old version
```suggestion
slug: /components/date-time/
```